ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/devkit/tasks/main.yml

```diff
@@ -0,0 +1,53 @@
+---
+################################
+# Adoptium devkit installation #
+################################
+
+- name: Check if devkit {{ devkit_version }} is installed
+  shell: /usr/local/devkit/{{ devkit_version }}/bin/gcc --version 2>&1 > /dev/null
+  failed_when: false
+  register: devkit_installed
+  changed_when: False
+  tags: devkit
+
+- name: Set filename of devkit tarball
+  set_fact:
+    devkit_filename: devkit-{{ devkit_version }}_{{ ansible_architecture }}-linux.tar.xz
+  when:
+    - devkit_installed.rc != 0
+  tags: devkit
+
+- name: Download {{ devkit_filename }}
+  get_url:
+    url: https://github.com/adoptium/devkit-binaries/releases/download/{{ devkit_version }}/{{  devkit_filename }}
+    dest: /tmp/devkit.tar.xz
+    force: no
+    mode: 0644
+    checksum: "sha256:{{ lookup('vars', 'csum_' + ansible_architecture) }}"
+  when:
+    - devkit_installed.rc != 0
+  tags: devkit
+
+# Perhaps this should have the `b01` removed ... Or use a symlink?
```

Comment:
Comment no longer relevant?

Reply:
No it's still relevant as I reverted a change I did yesterday - {{ devkit_version }} here still has the `bXX` in it so the comment could give people a "TODO" / food for thought